cmd/api-errors.go

```diff
@@ -593,6 +594,11 @@ var errorCodeResponse = map[APIErrorCode]APIError{
 		Description:    "The secret key is invalid.",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrAdminConfigNoQuorum: {
+		Code:           "XMinioAdminConfigNoQuorum",
+		Description:    "Minio Config doesn't have quorum.",
```

Comment:
Isn't this a bit cryptic? How about "Configuration update failed because a server quorum was not met"?